### PR TITLE
- Translation support

### PR DESCRIPTION
**Pull request by stijnhau**
- Translation support
  - Ducth translation complete

### CODE REVIEW

File: language/de_DE.po [hunk 1]
```diff
@@ -0,0 +1,56 @@
+msgid ""
+msgstr ""
+"Project-Id-Version: ZfcUser\n"
```
Comment (ojhaujjwal) [inline language/de_DE.po:3]: What`s this?

Comment (stijnhau) [inline language/de_DE.po:3]: The files where copied from my fprk of ZfcUser.
arent that important but you can change those if you want to the correct name.

Comment (ojhaujjwal) [inline language/de_DE.po:3]: Please change all these from ZfcUser to HtProfileImage and inform me!

**Comment by stijnhau** [comment]
Im closing this PR and i will create a new one later (probably this evenening or tomorrow)

**Comment by stijnhau** [comment]
The new issue has been created has id 18.

**Pull request closed without merging**